app/actions/index.js

```diff
@@ -1,16 +1,21 @@
 'use strict'
 
-import Dat from 'dat-node'
+import DatRaw from 'dat-node'
 import { encode } from 'dat-encoding'
 import { homedir } from 'os'
 import { clipboard, remote } from 'electron'
 import mirror from 'mirror-folder'
 import fs from 'fs'
 import promisify from 'util-promisify'
 import { basename } from 'path'
+import { createLock, createLocker } from 'flexLock'
 
 const dats = {}
+let pausedDats = {}
```

Comment:
why create an extra object for this? we don't need to keep track of this state

Reply:
It is easier to keep it in storage and it kinda disconnects the pause-state from the dats (referring to the discussion in the chat)